src/main/resources/config.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Do you expect this file to have non database related config as well? Just thinking from the file naming perspective